### PR TITLE
Comment button

### PR DESCRIPTION
**Pull request by john-li7**
Created activeThreadID in state to keep track of which feed level post has its thread open. Created a comment button on each post which will change the activeThreadID in state to the specific post's ID. In the PostsContainer file, conditional rendering logic was added to the rendering of all feed posts to check if the feed being rendered has the matching ID to the activeThreadID in state. If so, ThreadContainer dummy will be rendered. To uncomment line 32 which will render ThreadContainer with alias and post ID passed as props.
	
Also added createdat (to be updated to timestamp format) to display of posts. Minor updates to include alias as a property in state to begin implementation of alias feature.

### CODE REVIEW

File: client/components/Post.jsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 
-export default function Post({ title, body, styling }) {
-  console.log(styling);
+export default function Post({ title, body, datetime, feedPostID, styling, updateActiveThreadID }) {
```
Comment (Carthanial) [inline client/components/Post.jsx:3]: Is the title going to be switched to alias?

Comment (Carthanial) [inline client/components/Post.jsx:3]: Is the datetime the difference between createdat and invocation of now()?

File: client/components/PostForm.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ const mapStateToProps = (state) => {
     newPostTitle: state.posts.newPostTitle,
     newPostBody: state.posts.newPostBody,
     user: state.scratch.user,
+    // alias: state.posts.alias,
```
Comment (Carthanial) [inline client/components/PostForm.jsx:10]: As per your PR comment, there is no logic yet for this alias line, and that is why it is commented out correct?

File: client/components/PostForm.jsx [hunk 3]
```diff
@@ -17,7 +18,7 @@ const mapDispatchToProps = (dispatch) => {
     handleSubmit: (e, title, body, id) => {
```
Comment (Carthanial) [inline client/components/PostForm.jsx:18]: swap title with alias.

File: client/components/PostsContainer.jsx [hunk 4]
```diff
@@ -25,17 +23,36 @@ class PostsContainer extends Component {
...
-      return this.props.posts.posts.map((post, i) => (
-        <Post
+      return this.props.posts.posts.map((post, i) => {
+        return(
+        <div>
+          <Post
+          updateActiveThreadID={this.props.updateActiveThreadID}
           key={`Post ${i}`}
+          alias={post.alias}
           title={post.title}
```
Comment (Carthanial) [inline client/components/PostsContainer.jsx:45]: Waiting to take out title?

File: server/models/socialModels.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ const { Pool } = require('pg');
 
 const PG_URI =
   'postgres://yvigeiaa:Qpe4Y1GB8ZP-Rg-fKaLfbYM4Lin6Ce7J@ziggy.db.elephantsql.com:5432/yvigeiaa';
-
+// 'postgres://gilbfloq:VFX4UsWP_OJ43C2_jfOX2kRq-ktq0iXL@ziggy.db.elephantsql.com:5432/gilbfloq'
```
Comment (Carthanial) [inline server/models/socialModels.js:5]: This will  be switched when we merge the new backend into main.

File: client/components/Post.jsx (hunk 1 above)
Comment (john-li7) [reply client/components/Post.jsx:3]: Title will be switched in next PR.

Comment (john-li7) [reply client/components/Post.jsx:3]: Datetime is the same as createdat. Will be used to calculate the difference in Post file.

File: client/components/PostForm.jsx (hunk 2 above)
Comment (john-li7) [reply client/components/PostForm.jsx:10]: Correct, alias will be fully implement in next PR.

File: client/components/PostForm.jsx (hunk 3 above)
Comment (john-li7) [reply client/components/PostForm.jsx:18]: Will swap in next PR.

File: client/components/PostsContainer.jsx (hunk 4 above)
Comment (john-li7) [reply client/components/PostsContainer.jsx:45]: Will take out in next PR.

File: server/models/socialModels.js (hunk 5 above)
Comment (john-li7) [reply server/models/socialModels.js:5]: Confirmed.

File: client/components/Post.jsx (hunk 1 above)
Comment (unknownbreaker) [reply client/components/Post.jsx:3]: I'm good. @Carthanial ?

File: client/components/PostForm.jsx (hunk 2 above)
Comment (unknownbreaker) [reply client/components/PostForm.jsx:10]: I'm good. @Carthanial ?

File: client/components/PostForm.jsx (hunk 3 above)
Comment (unknownbreaker) [reply client/components/PostForm.jsx:18]: I'm good. @Carthanial ?

File: client/components/PostsContainer.jsx (hunk 4 above)
Comment (unknownbreaker) [reply client/components/PostsContainer.jsx:45]: I'm good. @Carthanial ?

**Pull request merged**